linalg/sundials.cpp

```diff
@@ -2184,6 +2188,43 @@ void KINSolver::SetMAA(int m_aa)
    }
 }
 
+void KINSolver::SetDamping(real_t beta)
+{
+   if (sundials_mem)
+   {
+      flag = KINSetDamping(sundials_mem, beta);
+      MFEM_ASSERT(flag == KIN_SUCCESS, "error in KINSetDamping()");
+   }
```

Comment:
I'm a little lost as to when one would expect these methods to be called when `sundials_mem == NULL` vs otherwise.  If I understand correctly, `sundials_mem` is set once `KINSolver::SetOperator` is called? If so, is the idea that if this method (and the other ones like it) are called before `SetOperator`, they do nothing?

Reply:
I included it only because another existing method had it 🤷

Reply:
What is the other method? My concern here is a silent failure, where the user calls `KINSolver::SetDamping` before calling `KINSolver::SetOperator` and then expects the value of `beta` to have an impact on the solve: the value will never be used because it is not stored locally (if I understand things correctly).